measure/src/main/kotlin/com/alliander/open/measure/SpecificUnits.kt

```diff
@@ -4,16 +4,60 @@
 
 package com.alliander.open.measure
 
+import com.alliander.open.measure.Current.Companion.ampere
 import com.alliander.open.measure.Energy.Companion.joule
 import com.alliander.open.measure.Power.Companion.watt
 import com.alliander.open.measure.Time.Companion.seconds
+import com.alliander.open.measure.Voltage.Companion.volt
 import java.math.BigDecimal
 
+@JvmName("powerTimesTime")
 operator fun Measure<Power>.times(duration: Measure<Time>): Measure<Energy> {
-    val p = this `as` watt
-    val dt = duration `as` seconds
+    val p = this `in` watt
+    val dt = duration `in` seconds
+    return Measure(p * dt, joule)
+}
+
+@JvmName("currentTimesVoltage")
+operator fun Measure<Current>.times(voltage: Measure<Voltage>): Measure<Power> {
+    val i = this `in` ampere
+    val v = voltage `in` volt
+    return Measure(i * v, watt)
+}
+
+@JvmName("voltageTimesCurrent")
+operator fun Measure<Voltage>.times(current: Measure<Current>): Measure<Power> {
+    val v = this `in` volt
+    val i = current `in` ampere
+    return Measure(i * v, watt)
+}
+
+@JvmName("powerDivVoltage")
+operator fun Measure<Power>.div(voltage: Measure<Voltage>): Measure<Current> {
+    val p = this `in` watt
+    val v = voltage `in` volt
+    return Measure(p / v, ampere)
+}
+
+@JvmName("powerDivCurrent")
+operator fun Measure<Power>.div(current: Measure<Current>): Measure<Voltage> {
+    val p = this `in` watt
+    val i = current `in` ampere
+    return Measure(p / i, volt)
+}
+
+class Voltage(suffix: String, ratio: BigDecimal = BigDecimal.ONE) : Units(suffix, ratio) {
```

Comment:
Voltage definition, I don't expect that we will use more then kiloVolts (our distribution grid is max 380kV)